build/lib/extensions.ts

```diff
@@ -313,7 +313,8 @@ const sqlBuiltInExtensions = [
 	'dacpac',
 	'schema-compare',
 	'resource-deployment',
-	'cms'
+	'cms',
+	'book'
 ];
```

Comment:
You'll also want to add in a .vscodeignore so that the VSIX doesn't end up having extra stuff in it - see [https://github.com/microsoft/azuredatastudio/blob/master/extensions/admin-tool-ext-win/.vscodeignore](https://github.com/microsoft/azuredatastudio/blob/master/extensions/admin-tool-ext-win/.vscodeignore) for an example